app-misc/terminal-colors/Manifest

```diff
@@ -1 +1,2 @@
 DIST terminal-colors-2.3.gz 5424 BLAKE2B e08635c0e989a430805159829b133f18b7c30959c945e4373a77442ac15628bb8bf3a5a2d14cd4f5705a5dd332297fa3d41611b52f56baaf9b1a882c26626714 SHA512 64fbfe766563c4c25bec23d1657187414f97b6f92e8a4a35337563210ef9873db495376676ef0a2f4e440ad77a6c1b43ee9c27fe04b34c2839eb4fffd5289a00
+DIST terminal-colors-3.0.2.tar.gz 5511 BLAKE2B e0d80997d1b95c88d00ec322a79519efce6f531ac66dc09462ada53eccdb6e3a0f150f6325d708475c1b50c3976f88291f24b500b32d350513712bd22624a061 SHA512 e00a9c6b3e16bb88cfa78f0fad76ce192641f693424aaf7c65d87dcf46324d3650cd553cdd53dae26e91d36dd57104cd27a173375771b79cc540e1cb4246258e
```

Comment:
```
!!! Fetched file: terminal-colors-3.0.2.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      152226
!!! Expected: 5511
```
That's a huge size difference. Please remove your local distfile, try again, and make sure this is the file that works with your ebuild as intended.